pym/portage/package/ebuild/_config/LocationsManager.py

```diff
@@ -298,6 +310,13 @@ def set_root_override(self, root_overwrite=None):
 		self.target_root = normalize_path(os.path.abspath(
 			self.target_root)).rstrip(os.path.sep) + os.path.sep
 
+		if self.sysroot != "/" and self.sysroot != self.target_root:
```

Comment:
Even if this assertion break things for existing build systems (like chromium os), I think it's acceptable for now. If necessary, we can find some way to accommodate them in the future.